public_html/index.html

```diff
@@ -35,28 +35,39 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
-            <a href="http://openwebgames.com" class="link">
+            <a href="https://metrics.mozilla.com/firefox-hardware-report" class="link">
               <div class="imageWrapper">
-                <img src="assets/img/tile-owg.jpg">
+                <img src="assets/img/tile-hardwarereport.jpg">
```

Comment:
I would remove the graphical text from this tile image - so it can be easily localised (and styled appropriately responsively). (I had a L10n system in place here for the original GDC site, but it was removed during the redesign of this site.)